### PR TITLE
MYLOC-362 API module reverse-geocode rounds co-ordinates to 3dp

### LINKED ISSUE #38: Reverse geocode should round coordinates to 3 decimal places
For cacheability and privacy reasons, the reverse geocode endpoint should round to 3 decimal places before a request is made.

Possibly here: https://github.com/BBC-Location-Services/locservices-core-js/blob/master/src/api.js#L190

### PR DESCRIPTION
**Pull request by tomcollins**
Fixes #38 

Not using Math.round due to the issues described in http://www.jacklmoore.com/notes/rounding-in-javascript/

### CODE REVIEW

**Comment by jrcryer** [comment]
Looks good to me - I'll await Darren's confirmation before merging.

**Comment by tomcollins** [comment]
I'm going to merge and put out a release as it is required for the UAS work in the UI project.

File: src/api.js [hunk 1]
```diff
@@ -36,6 +36,17 @@
   }
 
   /**
+   * Round a number to a specified pecision.
```
Comment (darrenlucas) [inline src/api.js]: Typo 'pecision' should be 'precision'

File: src/api.js [hunk 2]
```diff
@@ -36,6 +36,17 @@
...
+   *
+   * @param {Number} value - the number to round
+   * @param {Number} precision - the number of deimal places desired
+   * @returns {Number}
+   */
+  function roundNumber(value, precision) {
+    return Number(Math.round(value+'e'+precision)+'e-'+precision);
```
Comment (darrenlucas) [inline src/api.js]: Interesting use of exponential values to limit the precision.

Did you look at using Number.toPrecision() / Number.toFixed() ?

* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision

Comment (darrenlucas) [inline src/api.js]: I should have read the article you linked to :)

**Pull request merged**